moveit2_tutorials.repos

```diff
@@ -3,3 +3,8 @@ repositories:
     type: git
     url: https://github.com/PickNikRobotics/rviz_visual_tools
     version: foxy-devel
+
```

Comment:
```suggestion
```

Reply:
Just a small nit. We have not used space in the .repos file all over moveit2 so just for consistency. Otherwise looks good to me thanks a ton @facontidavide !